api/app.py

```diff
@@ -123,10 +123,13 @@ def course_equivalencies_search():
     query = request_body.get('query', "")
     programs = request_body.get('programs', [])
     unis = request_body.get('unis', [])
+    uni_search = request_body.get('uni_search', False)
         
     filters = []
-    if query is not None:
+    if query is not None and not uni_search:
         filters.append((UWCourse.name.like('%'+query+'%') | UWCourse.code.like('%'+query+'%')))
+    if query is not None and uni_search:   
+        filters.append((University.name.like('%'+query+'%')))
```

Comment:
For this endpoint 'query' is a uw course code or uw course name. Using query interchangeably to be the university name leads to the parameter becoming ambiguous. 

If you have the university name you can just send it to the Unis list. 

If you only have part of the university name then create another optional parameter to indicate the partial university name would be best.

Reply:
Oh my idea was that I passed in the `uni_seach` optional parameter to indicate that I'm in fact using a partial university name. It shouldn't interfere with the other uses of this endpoint because in my course search I'm explicitly passing in `uni_search`

Reply:
The ambiguity comes from what `query` is used for. In one case it is used for **UW Courses** and when ur boolean is `_true_ + query` is being used for a **University name.** 

I recommend not passing in a boolean to make the `query `parameter switch from being used to control the UW Course filter (line 130) to control University attributes (line 132). You should instead pass a parameter to control University attributes like `uni_filter `and then have a line like 

`filters.append((University.name.like('%'+uni_filter +'%')))`

Reply:
Resolved!